History timeline

Yiroh opened the pull request:
Added history of open source, fixed issue where open@RIT page did not immediately redirect you to the new Open@RIT website #188, added history tab to under the dropdown in about.

MoralCode commented:
@Nolski I can probably get to it next week if i cant get to it in the next 3 days

Yiroh commented:
Yeah this PR is ready for review, no rush though!

MoralCode commented:
@Yiroh the history of your repo has diverged from this one. a bunch of the commits in this PR are already in main it seems (maybe this is my fault for how i merged it, but also you were committing directly to main for your previous contribution in #190 so now things are a bit of a mess).


happy to help you rebase things so i can review this PR!

MoralCode commented on history.md:
```diff
@@ -0,0 +1,88 @@
+
+## OPEN@RIT:
+
+In 2020, the [Alfred P. Sloan Foundation](https://sloan.org/) provided [financial support](https://sloan.org/grant-detail/10076) for an effort led by the Rochester Institute of Technology’s Stephen Jacobs to design and launch an undergraduate-anchored program: [Open@RIT](https://openr.it/about/). 
```
while the classes (HFOSS .etc) are undergrad focussed, OPEN@RIT supports the entire RIT community.

MoralCode commented on history.md:
```diff
@@ -0,0 +1,88 @@
+## OPEN@RIT:
+
+In 2020, the [Alfred P. Sloan Foundation](https://sloan.org/) provided [financial support](https://sloan.org/grant-detail/10076) for an effort led by the Rochester Institute of Technology’s Stephen Jacobs to design and launch an undergraduate-anchored program: [Open@RIT](https://openr.it/about/). 
+This program was situated just under Rochester Institute of Technologies Vice President for Research, [Ryne Raffaelle](https://www.rit.edu/directory/rprsps-ryne-raffaelle) and has been the home of open source at RIT for the last four years. 
```
I'm noticing a few small things throughout this that feel all little off grammar wise, like "was situated" vs "is situated", or "the program now participants" in the next line that may warrant a larger scale check to make sure the whole post is okay

Yiroh replied:
Yeah, I swept through it twice and fixed quite a few typos.

MoralCode commented on history.md:
```diff
@@ -0,0 +1,88 @@
+*Adrian Edwards
+*Michael Nolan
+
+https://rocwiki.org/BarCamp_Rochester
```
do these links have context? it may be helpful to have them linked inline (if they aren't already) rather than chilling out at the bottom here

Yiroh replied:
The links were front the meeting notes with Michael Nolan, I did add them inline, but when we fixed the timeline history, this got re-added. Removing them.

MoralCode commented on history.md:
```diff
@@ -0,0 +1,88 @@
+
+### Individuals Information Contribution Credit:
+
+*axk4545
```
put spaces after the stars so markdown detects this as a bullet list and can format it properly

mansam commented on history.md:
```diff
@@ -0,0 +1,81 @@
+
+## FOSSBOX:
+
+Estimated to have started in 2010, Stephen Jacobs hired Remy DeCausemaker to run the open source program. Remy was initially from the Red Hat Corporation so the club had a great foundation.
```
> Remy was initially from the Red Hat Corporation so the club had a great foundation.

@decause was also an RIT alum.

MoralCode commented:
@Nolski you want a chance to review this pre-merge?

MoralCode commented on history.md:
```diff
@@ -50,39 +50,32 @@ The minor was brought about by additional courses offered by the School of Inter
 ## OPEN@RIT:
 
-In 2020, the [Alfred P. Sloan Foundation](https://sloan.org/) provided [financial support](https://sloan.org/grant-detail/10076) for an effort led by the Rochester Institute of Technology’s Stephen Jacobs to design and launch an undergraduate-anchored program: [Open@RIT](https://openr.it/about/). 
+In 2020, the [Alfred P. Sloan Foundation](https://sloan.org/) provided [financial support](https://sloan.org/grant-detail/10076) for an effort led by the Rochester Institute of Technology’s Stephen Jacobs to design and launch an official open source program: [Open@RIT](https://openr.it/about/). 
```
careful with the word "source". OPEN@RIT is more than just source code

MoralCode replied:
wait i was looking at an old version. disregard

MoralCode commented:
@jwflory want a chance to review this as well?

jwflory commented on _config.yml:
```diff
@@ -30,6 +30,7 @@ seo_keywords: >  # this symbol ignores newlines
   "Stephen Jacobs"
 twitter_account: "@RITMAGIC"
 url: "https://fossrit.github.io"
+open@ritURL: "https://openr.it/"
```
I am not a Jekyll expert but it is probably a good idea to avoid using symbols in variable names.

```suggestion
openRitUrl: "https://openr.it/"
```

MoralCode replied:
@Yiroh is this URL referred to in many places? I guess its nice to have this as a variable in case the site changes to using the one on the rit.edu domain, but it would be more compelling if it was actually needed due to linkink to OPEN@RIT from several places

MoralCode replied:
but yeah definitely remove the symbol

MoralCode replied:
seems like this may also have been intended to cover the link to OPEN@RIT under the "who we are" section of the homepage

Yiroh replied:
Yeah so I added that when I was trying to make the tab redirect to Open@RIT in tabs.yml, however I learned you could just type the link in and it'd work as intended so I can safely remove this.

jwflory commented on history.md:
```diff
@@ -0,0 +1,81 @@
+
+Stephen Jacobs, director of RIT's Lab for Technological Literacy (LTL) at the time, was interested in the program and acquired three OLPC XO 1.0 laptops in 2007 via the first "Give One, Get One" program. 
+In January 2009, Rochester Institute of Technology had an honors class that developed educational games for the OLPC program. That was the earliest documented birth of Open-Source @ RIT. 
+Open-source advocates, such as Karlie Robinson and David Nalley, were brought in by Professor Jacobs to support the class. 
```
```suggestion
Open Source advocates, such as Karlie Robinson and David Nalley, were brought in by Professor Jacobs to support the class.
```

Open Source in capitals because it is referring to a proper noun in this case: https://opensource.org/definition-annotated

jwflory commented on history.md:
```diff
@@ -0,0 +1,81 @@
+In January 2009, Rochester Institute of Technology had an honors class that developed educational games for the OLPC program. That was the earliest documented birth of Open-Source @ RIT. 
+Open-source advocates, such as Karlie Robinson and David Nalley, were brought in by Professor Jacobs to support the class. 
+Karlie Robinson introduced the students to the Sugar Labs' Math 4 initiative which was dedicated to supporting 4th grade curricula. 
+David Nalley, who was part of the Fedora community, donated 25 XO laptops to help students contribute to the Sugar initiative.
```
```suggestion
David Nalley, who was part of the [Fedora community](https://fedoraproject.org), donated 25 XO laptops to help students contribute to the Sugar initiative.
```

jwflory replied:
```suggestion
David Nalley, who was part of the [Fedora community](https://fedoraproject.org), donated 25 XO laptops to help students contribute to the Sugar initiative.
```

Just making it clear that we are not talking about hats. 🙂

jwflory commented on history.md:
```diff
@@ -0,0 +1,81 @@
+Open-source advocates, such as Karlie Robinson and David Nalley, were brought in by Professor Jacobs to support the class. 
+Karlie Robinson introduced the students to the Sugar Labs' Math 4 initiative which was dedicated to supporting 4th grade curricula. 
+David Nalley, who was part of the Fedora community, donated 25 XO laptops to help students contribute to the Sugar initiative.
+This ended the first iteration of the class, but the seminar class went on for several more iterations, with more incredible open-source advocates joining the picture.
```
```suggestion
This ended the first iteration of the class, but the seminar class went on for several more iterations, with more incredible Open Source advocates joining the picture.
```

jwflory commented on history.md:
```diff
@@ -0,0 +1,81 @@
+
+
+
+## FOSSBOX:
```
```suggestion
## FOSSBox
```

jwflory commented on history.md:
```diff
@@ -0,0 +1,81 @@
+
+## FOSSBOX:
+
+Estimated to have started in 2010, Stephen Jacobs hired Remy DeCausemaker (RIT alumni) to run the open-source program. Remy was initially from the Red Hat Corporation so the club had a great foundation.
```
```suggestion
Estimated to have started in 2010, Stephen Jacobs hired Remy DeCausemaker (RIT alumni) to run the Open Source program.
Remy was initially from the Fedora community so the club had a great foundation.
```

I am pretty sure @decause's chapter at Red Hat started well after his RIT chapter, but he was involved with Fedora from some time before Red Hat and I think before RIT too. I could be wrong, maybe he can help set the record straight here. 🙂

jwflory commented on history.md:
```diff
@@ -0,0 +1,81 @@
+## FOSSBOX:
+
+Estimated to have started in 2010, Stephen Jacobs hired Remy DeCausemaker (RIT alumni) to run the open-source program. Remy was initially from the Red Hat Corporation so the club had a great foundation.
+Initially, it was set up as a semi-student club, but more of an informal student group called the Free and Open-Source Software Box (FOSSBOX). It served as a 24-hour access space that was housed in the innovation center (also known as Fish Bowl). FOSSBOX used an IRC channel on Freenode to communicate when not in the designated space.
```
```suggestion
Initially, it was set up as a semi-student club, but more of an informal student group called the Free and Open Source Software Box (FOSSBox).
It served as a 24-hour access space that was housed in the Innovation Center (informally known as the Fish Bowl).
FOSSBox used an IRC channel on Freenode to communicate when not in the designated space.
```

jwflory commented on history.md:
```diff
@@ -0,0 +1,81 @@
+
+Estimated to have started in 2010, Stephen Jacobs hired Remy DeCausemaker (RIT alumni) to run the open-source program. Remy was initially from the Red Hat Corporation so the club had a great foundation.
+Initially, it was set up as a semi-student club, but more of an informal student group called the Free and Open-Source Software Box (FOSSBOX). It served as a 24-hour access space that was housed in the innovation center (also known as Fish Bowl). FOSSBOX used an IRC channel on Freenode to communicate when not in the designated space.
+As the first group to introduce hackathons to the student population, FOSSBOX's impact on RIT's culture continues as several hackathons are run every semester.
```
```suggestion
As the first group to introduce hackathons to the student population, the FOSSBox's impact on RIT's culture continues as several hackathons are run every semester.
```

jwflory commented on history.md:
```diff
@@ -0,0 +1,81 @@
+Estimated to have started in 2010, Stephen Jacobs hired Remy DeCausemaker (RIT alumni) to run the open-source program. Remy was initially from the Red Hat Corporation so the club had a great foundation.
+Initially, it was set up as a semi-student club, but more of an informal student group called the Free and Open-Source Software Box (FOSSBOX). It served as a 24-hour access space that was housed in the innovation center (also known as Fish Bowl). FOSSBOX used an IRC channel on Freenode to communicate when not in the designated space.
+As the first group to introduce hackathons to the student population, FOSSBOX's impact on RIT's culture continues as several hackathons are run every semester.
+FOSSBOX also ran several of their hackathons when they were active, such as [Election Night Hackathon](https://fossrit.github.io/events/2019/11/05/election-night-hackathon/) and Random Hacks of Kindness, as well as several others.
```
```suggestion
FOSSBox also ran several of their hackathons when they were active, such as [Election Night Hackathon](https://fossrit.github.io/events/2019/11/05/election-night-hackathon/) and Random Hacks of Kindness, as well as several others.
```

jwflory commented on history.md:
```diff
@@ -0,0 +1,81 @@
+Initially, it was set up as a semi-student club, but more of an informal student group called the Free and Open-Source Software Box (FOSSBOX). It served as a 24-hour access space that was housed in the innovation center (also known as Fish Bowl). FOSSBOX used an IRC channel on Freenode to communicate when not in the designated space.
+As the first group to introduce hackathons to the student population, FOSSBOX's impact on RIT's culture continues as several hackathons are run every semester.
+FOSSBOX also ran several of their hackathons when they were active, such as [Election Night Hackathon](https://fossrit.github.io/events/2019/11/05/election-night-hackathon/) and Random Hacks of Kindness, as well as several others.
+The original Humanitarian Free and Open-Source Software (HFOSS) seminar class was accredited and students in that class commonly got introduced and involved in the student group as well.
```
```suggestion
The original Humanitarian Free and Open Source Software (HFOSS) seminar class was accredited and students in that class commonly got introduced and involved in the student group as well.
```

jwflory commented on history.md:
```diff
@@ -0,0 +1,81 @@
+As the first group to introduce hackathons to the student population, FOSSBOX's impact on RIT's culture continues as several hackathons are run every semester.
+FOSSBOX also ran several of their hackathons when they were active, such as [Election Night Hackathon](https://fossrit.github.io/events/2019/11/05/election-night-hackathon/) and Random Hacks of Kindness, as well as several others.
+The original Humanitarian Free and Open-Source Software (HFOSS) seminar class was accredited and students in that class commonly got introduced and involved in the student group as well.
+Remy and FOSSBOX started the [course-to-coop life cycle](https://opensource.com/education/10/3/course-co-op-lifecycle-openinnovationrit) where students in open-source would receive co-op offers from companies that were funding and interested in open-source at RIT. This cycle repeated as those students graduated and got full-time offers and referred those who were in the student club for co-op pos ...
```
```suggestion
Remy and FOSSBox started the [course-to-coop life cycle](https://opensource.com/education/10/3/course-co-op-lifecycle-openinnovationrit) where students in Open Source would receive co-op offers from companies that were funding and interested in Open Source at RIT.
This cycle repeated as those students graduated and got full-time offers and referred those who were in the student club for co-op positions.
```

jwflory commented on history.md:
```diff
@@ -0,0 +1,81 @@
+FOSSBOX also ran several of their hackathons when they were active, such as [Election Night Hackathon](https://fossrit.github.io/events/2019/11/05/election-night-hackathon/) and Random Hacks of Kindness, as well as several others.
+The original Humanitarian Free and Open-Source Software (HFOSS) seminar class was accredited and students in that class commonly got introduced and involved in the student group as well.
+Remy and FOSSBOX started the [course-to-coop life cycle](https://opensource.com/education/10/3/course-co-op-lifecycle-openinnovationrit) where students in open-source would receive co-op offers from companies that were funding and interested in open-source at RIT. This cycle repeated as those students graduated and got full-time offers and referred those who were in the student club for co-op pos ...
+FOSSBOX may have been an informal student group at first but it grew to the point where a full-time staff member was managing FOSSBOX and undergraduate students had part-time paid roles doing the secretarial work for FOSSBOX, both of which were funded by a yearly payment by Red Hat Corporation.
```
```suggestion
FOSSBox may have been an informal student group at first, but it grew to the point where a full-time staff member was managing FOSSBox and undergraduate students had part-time paid roles doing the secretarial work for FOSSBox, both of which were funded by a yearly payment by Red Hat.
```

jwflory commented on history.md:
```diff
@@ -0,0 +1,81 @@
+The original Humanitarian Free and Open-Source Software (HFOSS) seminar class was accredited and students in that class commonly got introduced and involved in the student group as well.
+Remy and FOSSBOX started the [course-to-coop life cycle](https://opensource.com/education/10/3/course-co-op-lifecycle-openinnovationrit) where students in open-source would receive co-op offers from companies that were funding and interested in open-source at RIT. This cycle repeated as those students graduated and got full-time offers and referred those who were in the student club for co-op pos ...
+FOSSBOX may have been an informal student group at first but it grew to the point where a full-time staff member was managing FOSSBOX and undergraduate students had part-time paid roles doing the secretarial work for FOSSBOX, both of which were funded by a yearly payment by Red Hat Corporation.
+The FOSSBOX community operated for 9 years, collaborating with organizations in the local area such as [Hacks/Hackers](https://www.hackshackers.com/), [BarCampROC](https://rocwiki.org/BarCamp_Rochester), and the student club [CodeRIT](https://github.com/codeRIT).
```
```suggestion
The FOSSBox community operated until approximately 2015, collaborating with organizations in the local area such as [Hacks/Hackers](https://www.hackshackers.com/), [BarCampROC](https://rocwiki.org/BarCamp_Rochester), and the student club [CodeRIT](https://github.com/codeRIT).
```

jwflory commented on history.md:
```diff
@@ -0,0 +1,81 @@
+
+
+
+## LibreCorps:
```
```suggestion
## LibreCorps
```

jwflory commented on history.md:
```diff
@@ -0,0 +1,81 @@
+## LibreCorps:
+
+Founded in 2012, LibreCorps' first purpose was as a co-op program after developing educational games under the 'One Laptop Per Child' program.
+LibreCorps serves as an experiential learning program where students are deployed to work on real-world open-source software and data projects, supporting faculty partners with direct software development as well as community management, documentation, design, and other forms of non-code contribution.
```
```suggestion
LibreCorps serves as an experiential learning program where students are deployed to work on real-world Open Source software and data projects, supporting faculty partners with direct software development as well as community management, documentation, design, and other forms of non-code contribution.
```

jwflory commented on history.md:
```diff
@@ -0,0 +1,81 @@
+
+Founded in 2012, LibreCorps' first purpose was as a co-op program after developing educational games under the 'One Laptop Per Child' program.
+LibreCorps serves as an experiential learning program where students are deployed to work on real-world open-source software and data projects, supporting faculty partners with direct software development as well as community management, documentation, design, and other forms of non-code contribution.
+RIT LibreCorps has been partnered intermittently with the UNICEF Office of Innovation since 2014.
```
```suggestion
RIT LibreCorps has been partnered intermittently with the [UNICEF Office of Innovation](https://www.unicef.org/innovation/) since 2014.
```

jwflory commented on history.md:
```diff
@@ -0,0 +1,81 @@
+
+
+
+## FOSS@RIT:
```
TBH, I don't think there was ever a "FOSS@RIT" community. It was always FOSS Box or FOSS@MAGIC. FOSS@RIT was just the GitHub organization that everything circled around.

MoralCode replied:
news to me haha, i imagine FOSS@RIT started as just a generic center/unchanging name for the rest of this stuff to revolve around?

jwflory replied:
@MoralCode The FOSS Box and FOSS@RIT were equivocally the same thing.

jwflory commented on history.md:
```diff
@@ -0,0 +1,81 @@
+
+## FOSS@MAGIC:
+
+When [Magic Spell Studio](https://www.rit.edu/magic/magic-spell-studios) was conceptualized, this organization was established to prioritize game development, which was where open-source got started on this campus.
```
```suggestion
When MAGIC was conceptualized in the Simone Center for Student Innovation, it was established to prioritize game development.
However, many of the same drivers and organizers of FOSS@RIT were deeply tied into MAGIC, which was how the Open Source was reborn on the RIT campus.
```

Well, it did not really start there. 🙂

jwflory commented on history.md:
```diff
@@ -0,0 +1,81 @@
+## FOSS@MAGIC:
+
+When [Magic Spell Studio](https://www.rit.edu/magic/magic-spell-studios) was conceptualized, this organization was established to prioritize game development, which was where open-source got started on this campus.
+This organization was managed by [Andrew Phelps](https://professorandrewphelps.net/) in 2015. Since the magic building was not completed at the time, they were relocated to the innovation center, in the exact location that FOSSBOX initially used. 
```
```suggestion
MAGIC was managed and led by [Andrew Phelps](https://professorandrewphelps.net/) in 2015.
FOSS@MAGIC was headquartered in the Innovation Center, albeit without a dedicated physical space like the FOSSBox originally had.
```

jwflory commented on history.md:
```diff
@@ -0,0 +1,81 @@
+
+When [Magic Spell Studio](https://www.rit.edu/magic/magic-spell-studios) was conceptualized, this organization was established to prioritize game development, which was where open-source got started on this campus.
+This organization was managed by [Andrew Phelps](https://professorandrewphelps.net/) in 2015. Since the magic building was not completed at the time, they were relocated to the innovation center, in the exact location that FOSSBOX initially used. 
+FOSS Hours were introduced, which helped increase the involvement of students and other interested parties. However, shortly after, Remy DeCausemaker left Rochester Institute of Technology to go work for Twitter, which made the open-source community lose its growing momentum.
```
```suggestion
FOSS Hours were introduced, which helped increase the involvement of students and other interested parties.
However, shortly after, Remy DeCausemaker left the Rochester Institute of Technology in 2015 to go work for Red Hat, which made the RIT Open Source community lose its growing momentum.
```

jwflory commented on history.md:
```diff
@@ -0,0 +1,81 @@
+
+
+
+## OPEN@RIT:
```
```suggestion
## Open@RIT
```

jwflory commented on history.md:
```diff
@@ -0,0 +1,81 @@
+## OPEN@RIT:
+
+In 2020, the [Alfred P. Sloan Foundation](https://sloan.org/) provided [financial support](https://sloan.org/grant-detail/10076) for an [open work](https://openworkdefinition.com/) organization led by the Rochester Institute of Technology’s Stephen Jacobs: [Open@RIT](https://openr.it/about/). 
+This program was situated just under Rochester Institute of Technologies Vice President for Research, [Ryne Raffaelle](https://www.rit.edu/directory/rprsps-ryne-raffaelle) and has been the home of open-source at RIT for the last four years. 
```
```suggestion
This program was situated just under Rochester Institute of Technology Vice President for Research, [Ryne Raffaelle](https://www.rit.edu/directory/rprsps-ryne-raffaelle) and has been the home of Open Source at RIT since 2020.
```

"Past four years" will get outdated quickly, best to use absolute dates instead of relative dates.

jwflory commented on history.md:
```diff
@@ -0,0 +1,81 @@
+
+In 2020, the [Alfred P. Sloan Foundation](https://sloan.org/) provided [financial support](https://sloan.org/grant-detail/10076) for an [open work](https://openworkdefinition.com/) organization led by the Rochester Institute of Technology’s Stephen Jacobs: [Open@RIT](https://openr.it/about/). 
+This program was situated just under Rochester Institute of Technologies Vice President for Research, [Ryne Raffaelle](https://www.rit.edu/directory/rprsps-ryne-raffaelle) and has been the home of open-source at RIT for the last four years. 
+The program now participates in research projects, receives funding from grants, provides other organizations open source consultation, and hosts town halls for the open-source community on campus. 
```
```suggestion
Open@RIT participates in research projects, receives funding from grants, provides open source community consultation to faculty and staff, and hosts town halls for the Open Source community on campus.
```

jwflory commented on history.md:
```diff
@@ -0,0 +1,81 @@
+
+One Laptop Per Child ([OLPC](https://laptop.org/)) was proposed by MIT in 2005 to [fight the gap](https://laptop.org/aboutolpc/) in access to technology in education for underprivileged children.
+
+Stephen Jacobs, director of RIT's Lab for Technological Literacy (LTL) at the time, was interested in the program and acquired three OLPC XO 1.0 laptops in 2007 via the first "Give One, Get One" program. 
```
```suggestion
[Prof. Stephen Jacobs](https://www.rit.edu/directory/sxjics-stephen-jacobs), director of RIT's Lab for Technological Literacy (LTL) at the time, was interested in the program and acquired three OLPC XO 1.0 laptops in 2007 via the first "Give One, Get One" program.
```

MoralCode commented on history.md:
```diff
@@ -0,0 +1,93 @@
+
+
+
+### Individuals Information Contribution Credit:
```
@jwflory u want to add yourself here? or maybe just link to this PR?

MoralCode commented on history.md:
```diff
@@ -0,0 +1,93 @@
+
+FOSS@RIT is the name of the GitHub organization for all things open-source at Rochester Institute of Technology, however, the organization isn't very official. 
+During the period of renaming from FOSSBOX to FOSS@MAGIC, this was what you would call the Open-Source hub, though interchangeable with the FOSS@MAGIC name.
+The FOSS@RIT community was full of people with an interest in free and open-source software at RIT. At the time, FOSS@RIT had the initial purpose of keeping FOSSBOX alive.
```
i wonder if this could use more. it seems up cut off fairly abruptly with this sentence on what FOSS@RIT was. maybe we could give it a bit of a new name/purpose? especially with the reboot (ish) of FOSS-hours and my interest in promoting student projects with the discord server

MoralCode replied:
this may also make a good little mission statement for the "who we are" section of the homepage maybe

Yiroh commented:
With the newest commit, I removed that URL (was not being referenced anywhere, I forgot to remove it like I said when I was figuring out how to make the link for the tab to Open@RIT, removed the FOSS@RIT section but expanded onto the FOSSBox section to talk about how the Github ORG and FOSSBox were linked. 

Let me know if that is the correct information we wish to display.

MoralCode commented:
seems good! anyone else who should add their review? maybe SJ?

MoralCode commented:
history was kinda a mess so i essentially rebased this
